hll/src/main/java/io/druid/hll/HyperLogLogCollector.java

```diff
@@ -294,6 +298,11 @@ protected ByteBuffer getStorageBuffer()
     return storageBuffer;
   }
 
+  public void add(String rawValue)
```

Comment:
The collector is supposed to be hash-function agnostic, so this is a bit out of place.